appservice/src/SiteClient.ts

```diff
@@ -259,4 +259,10 @@ export class SiteClient {
     public async getPublishingUser(): Promise<User> {
         return await this._client.getPublishingUser({});
     }
+
+    public async getWebJobs(): Promise<WebJobCollection> {
+        return this.slotName ?
+            await this._client.webApps.listWebJobsSlot(this.resourceGroup, this.siteName, this.slotName) :
+            await this._client.webApps.listWebJobs(this.resourceGroup, this.siteName);
```

Comment:
Do you need to add a listWebJobsNext? Does that exist?

Reply:
Does not exist

Reply:
Ugh, sorry this is incorrect.  It wasn't showing up before, but I see it in the API.